tests/test_dataset.py

```diff
@@ -295,10 +296,21 @@ def test_Dataset_auto_foreign_keys(tmpdir):
 
 def test_Dataset_from_data_empty_file(tmpdir):
     write_text(str(tmpdir / 'values.csv'), '')
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match='empty data file'):
         Dataset.from_data(str(tmpdir / 'values.csv'))
 
 
+@pytest.mark.parametrize('cls, expected', [
+    (Dataset, ParallelText),
```

Comment:
The default should be `Wordlist`, when encountering a file named `forms.csv`.

Reply:
Done, pushed.